api/src/main/resources/api/service_openapi.yaml

```diff
@@ -364,9 +364,21 @@ components:
           type: array
           items:
             type: string
+        orderByAttribute:
```

Comment:
This should be an array of attribute/direction pairs for flexibility since it's easy to support in SQL.

Reply:
Good suggestion, thanks. Do we have a use case for multiple order-by criteria currently? If not, then l'd like to address it in a follow-on PR.

Reply:
We don't right now but it's pretty common. Just suggesting it while you're defining the API. Putting it off for now is fine though since we don't immediately need it.

Reply:
Sounds good, I'll add it to the list of future enhancements then. Thanks